net-firewall/ferm/ferm-2.3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils systemd
+
+DESCRIPTION="Command line util for managing firewall rules"
+HOMEPAGE="http://ferm.foo-projects.org/"
+SRC_URI="http://ferm.foo-projects.org/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
On the official page they have: Licensed under the GPLv2. They have a link to the license: http://ferm.foo-projects.org/download/COPYING. Licenses are not my field of expertise, if you say so I can change it.